tests/eve-flow-vlan-02/test.yaml

```diff
@@ -0,0 +1,26 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
```

Comment:
`HAVE_LIBJANSSON` is implied by `min-version: 7` ;-)

Reply:
:-)